src/applications/personalization/profile360/components/ContactInformation.jsx

```diff
@@ -67,7 +65,7 @@ export default class ContactInformation extends React.Component {
             <div>
               <h3>How do I update the email I use to sign in to VA.gov?</h3>
               <a
-                href={accountManifest.rootUrl}
+                href="/profile"
```

Comment:
should this actually go to `profile/account-security`?

Reply:
The user should be redirected to account security when they hit the profile root url and are not logged in, but I will make the change you suggested for clarity 👍